lib/mhv_ac/registration_form.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/models/form'
+
+module MHVAC
+  class RegistrationForm < Common::Form
+    attribute :icn, String
```

Comment:
@jcmeloni MHV allows quite literally anyone, even me, to create an account. They also support veteran beneficiaries, va employees who are not veterans, etc. We currently support LOA3 for just veterans is my understanding. We also require LOA3 for using MHV based services. So perhaps we can show an error message "using these services requires LOA3" or something to that effect. We should not do account creation for users that are not LOA3. But then, to future proof our logic, it's unclear to me how we would do that... 

Let's say in 6 months we support Veteran beneficiaries on LOA3, how would we distinguish veteran from veteran beneficiary? Can we make this distinction today? If so, maybe we can add additional logic for the default values on those booleans to future proof.